### ISSUE TITLE
Availability of training/fine-tuning scripts

### ISSUE THREAD

**Issue by roman-bushuiev** [agemagician/Ankh#37]
Hi! Are there any publicly available training/fine-tuning scripts? I would like to fine-tune the pre-trained Ankh on a custom dataset of protein sequences using the masked setup described in section 4.6.1 of the manuscript. It would be wonderful if I could utilize your code, but unfortunately, I couldn't find it anywhere. Thank you!

**Comment by hazemessamm** [comment]
Hello, sorry for the late reply, they should be available soon, we will announce it as soon as it becomes ready.
